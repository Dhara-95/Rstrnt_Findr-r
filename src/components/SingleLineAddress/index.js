import React from "react";

import { formatAddress } from "../../utils/formatAddress";

const SingleLineAddress = (props) => {
  const { unformattedAddress } = props;

  return <span className="text-grey">{formatAddress(unformattedAddress)}</span>;
};

export { SingleLineAddress };
