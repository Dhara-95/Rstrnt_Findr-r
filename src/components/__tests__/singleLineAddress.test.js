import React from "react";
import renderer from "react-test-renderer";

import { SingleLineAddress } from "../SingleLineAddress";

it("displays the Restaurants address correctly formatted", () => {
  const tree = renderer
    .create(
      <SingleLineAddress
        unformattedAddress={"55 Manor Road\nLondon W13 0LJ\nUnited Kingdom"}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
