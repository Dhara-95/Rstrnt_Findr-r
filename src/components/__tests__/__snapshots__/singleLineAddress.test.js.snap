// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Restaurants address correctly formatted 1`] = `
<span
  className=" restaurant-address text-grey"
>
  55 Manor Road, London W13 0LJ, United Kingdom
</span>
`;
